assets/settings/default.json

```diff
@@ -596,6 +596,8 @@
   // 4. Save when idle for a certain amount of time:
   //     "autosave": { "after_delay": {"milliseconds": 500} },
   "autosave": "off",
+  // Whether to persist editor selections and restore them on restart.
+  "persist_selections": true,
```

Comment:
Perhaps we could reuse `restore_on_startup`? We'd have to adjust the shape of that setting a bit, but as we start persisting more and more stuff, it may be beneficial to have it all under one setting key.